src/clients/inst_autosetup.rb

```diff
@@ -188,6 +188,12 @@ def main
           ["Import", Ops.get_map(Profile.current, "add-on", {})]
         )
         Call.Function("add-on_auto", ["Write"])
+        # The entry "kexec_reboot" in the Product description can be set
+        # by the AutoYaST configuration setting (general/forceboot) and should
+        # not be reset by any other Product description file.
+        # So we set it here again.
```

Comment:
And what happens if forceboot is not specified at all in the profile? Should we set it to false or should it left it untouched?